com.gooseberrydevelopment.pinepods.yml

```diff
@@ -0,0 +1,41 @@
+id: com.gooseberrydevelopment.pinepods
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: app
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: pinepods-client
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/madeofpendletonwool/PinePods/releases/download/0.7.1/Pinepods_0.7.1_amd64.deb
+        sha256: 0638853425d8831ec67aef59ed84ae55b55e409110fda269f9fdb90dd817e83c
+        only-arches: [x86_64]
+        dest-filename: pinepods.deb
+      - type: file
+        url: https://github.com/madeofpendletonwool/PinePods/releases/download/0.7.1/Pinepods_0.7.1_arm64.deb
+        sha256: <arm64_sha256_here>
```

Comment:
missing sha

Reply:
Correct sha for current build has been added